README mwmichelpach1.md

```diff
@@ -1,5 +1,12 @@
 # Cloudera Director Public Scripts
 
+## Contributing Code
+
+In order to contribute code you must sign one of two agreements. If you are choosing to contribute on your own behalf then sign the ICLA agreement (https://github.com/cloudera/livy/wiki/Individual-Contributor-License-Agreement-(ICLA). If you are contributing on behalf of an organization or a company you must sign the CCLA agreement (https://github.com/cloudera/livy/wiki/Corporate-Contributor-License-Agreement-(CCLA). 
```

Comment:
An extra closing parenthesis is needed after "(ICLA)" and another after "(CCLA)".